helper_scripts/cosmic/setup_cloudstackOps.sh

```diff
@@ -1,39 +0,0 @@
-#!/bin/bash
```

Comment:
Please keep this file as it is used for CloudStackOps toolbox. Not related to Marvin change.